platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -145,27 +145,42 @@ event_processor:
   callback_api_request:
     enabled: true
 
-# The list of clients for the Anchor server to safely communicate with the outside client domains.
+# The list of clients for the Anchor server to safely communicate with the outside wallet servers or clients.
 # Each item in the list may contain the following fields:
-#   - domain: the domain of the client
-#   - type: the type of the client. The supported values are `custodial` and `non-custodial`
-#   - public_key: the public key of the client. This must match the public key of the TOML file of the client domain
-#   - url: the URL of the client's callback API endpoint
+#   - type: `custodial` or `noncustodial`
+#
+#   If the type is `custodial`,
+#   - account: (required) the custodial account of the client. If the `account` of the transaction is the same as this
+#              account, the callback will be activated.
+#   - url: (required) the URL of the client's callback API endpoint
```

Comment:
Lets update the comments here to use the `callback_url` attribute name.

Reply:
Thanks for catching this. Updated.